bloom_site/lib/bloom_site_web/components/bloom_button.ex

```diff
@@ -0,0 +1,46 @@
+
+defmodule BloomSiteWeb.Components.BloomButton do
```

Comment:
If it is literally the core_components button but more, should this be called Button and just add an instruction on installation to remove `def button/1` from the consuming apps core_components?

Reply:
Good point. I will implement that :)

Reply:
It's more that the core_components are already imported in a new phoenix app and exports a component called button/1 so without telling the user to change it then they'd get a clash

Reply:
Yeah no my question was more a thought. I implemented the renaming and fixed a keyboard control issue with the styling :)